src/api/bucket.cc

```diff
@@ -30,8 +30,12 @@ Bucket::Bucket(const std::string &initial_name,
 
   if (IsBucketNameTooLong(name_)) {
     id_.as_int = 0;
+    throw std::length_error("Bucket name is too long (max length 256).");
```

Comment:
The max length message should use `kMaxBucketNameSize` instead of hardcoding `256`.